internal/migrate/stage_contract.go

```diff
@@ -20,35 +20,42 @@ package migrate
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"strings"
 
+	"github.com/logrusorgru/aurora"
 	"github.com/manifoldco/promptui"
-	"github.com/onflow/cadence"
 	"github.com/onflow/cadence/runtime/common"
-	"github.com/onflow/contract-updater/lib/go/templates"
-	flowsdk "github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk"
 	"github.com/onflow/flowkit/v2"
 	"github.com/onflow/flowkit/v2/output"
-	"github.com/onflow/flowkit/v2/transactions"
+	"github.com/onflow/flowkit/v2/project"
 	"github.com/spf13/cobra"
 
-	internaltx "github.com/onflow/flow-cli/internal/transactions"
-
 	"github.com/onflow/flow-cli/internal/command"
 )
 
+type stagingResults struct {
+	Results       map[common.AddressLocation]stagingResult
+	prettyPrinter func(err error, location common.Location) string
+}
+
+var _ command.ResultWithExitCode = &stagingResults{}
+
 var stageContractflags struct {
-	SkipValidation bool `default:"false" flag:"skip-validation" info:"Do not validate the contract code against staged dependencies"`
+	All            bool     `default:"false" flag:"all" info:"Stage all contracts"`
+	Accounts       []string `default:"" flag:"account" info:"Accounts to stage the contract under"`
+	SkipValidation bool     `default:"false" flag:"skip-validation" info:"Do not validate the contract code against staged dependencies"`
 }
 
 var stageContractCommand = &command.Command{
 	Cmd: &cobra.Command{
-		Use:     "stage-contract <CONTRACT_NAME>",
-		Short:   "stage a contract for migration",
-		Example: `flow migrate stage-contract HelloWorld`,
-		Args:    cobra.MinimumNArgs(1),
+		Use:   "stage-contract [contract names...]",
```

Comment:
Is it too late to change this to just `stage`?

Reply:
I think so, I don't think it would be wise to break functionality right now.